package.json

```diff
@@ -76,6 +76,7 @@
     "posttest": "eslint --ignore-path .gitignore . && npm audit",
     "prepack": "oclif-dev manifest && oclif-dev readme && npm shrinkwrap && git checkout -- package-lock.json",
     "test": "nyc --check-coverage --lines 90 --reporter=html --reporter=text mocha --forbid-only \"test/**/*.test.js\"",
-    "version": "oclif-dev readme && git add README.md"
+    "version": "oclif-dev readme && git add README.md",
+    "postinstall": "node welcome.js"
```

Comment:
Can this run a CLI command? Like `bin/run welcome` which is a hidden command that checks for a project and builds the welcome message if not found?

Reply:
Actually I'm fine with this as-is